f5/bigip/tm/ltm/lsnpool.py

```diff
@@ -0,0 +1,94 @@
+# coding=utf-8
+#
+#  Copyright 2014-2016 F5 Networks Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""BIG-IP® Local Traffic Manager™ (LTM®) LSN pool module.
+
+REST URI
+    ``http://localhost/mgmt/tm/ltm/lsn-pool``
+    ``http://localhost/mgmt/tm/ltm/lsn-log-profile``
+
+GUI Path
+    ``Carrier Grade NAT --> LSN Pools``
+    ``Carrier Grade NAT --> Logging Profiles --> LSN``
+
+REST Kind
+    ``tm:ltm:lsn-pool:*``
+    ``tm:ltm:lsn-log-profile:*``
+"""
+
+
+from f5.bigip.resource import Collection
+from f5.bigip.resource import Resource
+
+
+class LSNPools(Collection):
```

Comment:
This class can be named `Lsn_Pools` to avoid the use of `_format_resource_name` later. The underscore is automatically replaced by a hyphen.

Be sure to reflect the name change of the class throughout the rest of this PR though